web/src/components/apps/AppVersionHistory.jsx

```diff
@@ -637,7 +631,9 @@ class AppVersionHistory extends Component {
     // This is kinda hacky. This finds the equivalent downstream version because the midstream
     // version type does not contain metadata like version label or release notes.
     const currentMidstreamVersion = versionHistory.find(version => version.parentSequence === app.currentVersion.sequence) || app.currentVersion;
-
+    const olderVersions = versionHistory.filter((i, idx) => idx !== 0);
```

Comment:
what does this do?

Reply:
why index 0?

Reply:
filters out everything after the first or latest one since we pin that one to the top of the list. "olderVersions" is probably the wrong name, "otherAvailableVersions" is probably better